lib/Serialization/Deserialization.cpp

```diff
@@ -2836,8 +2836,30 @@ ModuleFile::getDeclCheckedImpl(DeclID DID, Optional<DeclContext *> ForcedContext
     for (TypeID dependencyID : dependencyIDs) {
       auto dependency = getTypeChecked(dependencyID);
       if (!dependency) {
+        // Stored properties in classes still impact class object layout because
+        // their offset is computed and stored in the field offset vector.
+        DeclDeserializationError::Flags flags;
+        
+        if (!isStatic) {
+          switch ((StorageKind)storageKind) {
```

Comment:
Just to be sure, what kind do we get for `@NSManaged` properties?

Reply:
Those ought to show up as computed. Just to be sure, I added an unimportable `@NSManaged` property to the IRGen test, and it didn't appear to disturb the vtable layout.